schema_based_on_diagram.sql

```diff
@@ -0,0 +1,72 @@
+
+CREATE TABLE patients (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  UNIQUE (id),
+  name VARCHAR NOT NULL,
+  date_of_birth DATE NOT NULL
+);
+
+CREATE TABLE treatments (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  UNIQUE (id),
+  type VARCHAR NOT NULL,
+  name VARCHAR NOT NULL
+);
+
+CREATE TABLE medical_histories (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id BIGINT REFERENCES patients(id),
+  treatment_id BIGINT REFERENCES treatments(id),
```

Comment:
- [ ] Great job so far 👍 There is no `treatment_id` in `medical_histories` as that particular association is being handled by the `medical_histories_treatment`. Kindly rectify this issue 👍

Reply:
@Kingstalux, Thanks for the kind review. I have made changes and would really appreciate it if you check my changes before I submit my project for review again.

Reply:
Hello @topeogunleye your good to go 👍